app/k9mail/src/main/java/com/fsck/k9/auth/OAuthConfigurations.kt

```diff
@@ -21,7 +21,7 @@ fun createOAuthConfigurationProvider(): OAuthConfigurationProvider {
             listOf("imap.gmail.com", "imap.googlemail.com", "smtp.gmail.com", "smtp.googlemail.com") to googleConfig,
             listOf("imap.mail.yahoo.com", "smtp.mail.yahoo.com") to OAuthConfiguration(
                 clientId = BuildConfig.OAUTH_YAHOO_CLIENT_ID,
-                scopes = listOf("mail-w"),
+                scopes = listOf("openid"),
```

Comment:
https://developer.yahoo.com/sign-in-with-yahoo/#step-two

It works but the token authed can't be used to fetch mail.

Reply:
Use `mail-r` with the error: Invalid scope